src/helpers.js

```diff
@@ -92,6 +92,10 @@ function getWindowFromNode(node) {
     throw new Error(
       `It looks like you passed a Promise object instead of a DOM node. Did you do something like \`fireEvent.click(screen.findBy...\` when you meant to use a \`getBy\` query \`fireEvent.click(screen.getBy...\`, or await the findBy query \`fireEvent.click(await screen.findBy...\`?`,
     )
+  } else if (node instanceof Array) {
```

Comment:
```suggestion
  } else if (Array.isArray(node)) {
```

See http://web.mit.edu/jwalden/www/isArray.html